common/lib/xmodule/xmodule/templates/problem/numericalresponse.yaml

```diff
@@ -2,66 +2,61 @@
 
-       >>Enter the number of fingers on a human hand<<
-       = 5
+        >>How many miles away from Earth is the sun? Use scientific notation to answer.<<
 
-       [explanation]
-       Pi, or the the ratio between a circle's circumference to its diameter, is an irrational number known to extreme precision. It is value is approximately equal to 3.14.
+        = 9.3*10^6 
+        or= 9.296*10^6 
 
-       Although you can get an exact value by typing 502*9 into a calculator, the result will be close to 500*10, or 5,000. The grader accepts any response within 15% of the true value, 4518, so that you can use any estimation technique that you like.
+        >>The square of what number is -100?<<
 
-       If you look at your hand, you can count that you have five fingers.
-       [explanation]
-data: |
-   <problem>
-   <p>
-   A numerical input problem accepts a line of text input from the
-   student, and evaluates the input for correctness based on its
-   numerical value.  
-   </p>
-
-   <p>
-   The answer is correct if it is within a specified numerical tolerance
-   of the expected answer.
-   </p>
-   <script type="loncapa/python">
-   computed_response = 502*9
-   </script>
+        = 10*i 
 
-   <p>Enter the numerical value of Pi:
-       <numericalresponse answer="3.14159">
-           <responseparam type="tolerance" default=".02" />
-           <formulaequationinput label="Enter the numerical value of Pi" />
-       </numericalresponse>
-   </p>
-   <p>Enter the approximate value of 502*9:
-       <numericalresponse answer="$computed_response">
-           <responseparam type="tolerance" default="15%"/>
-           <formulaequationinput label="Enter the approximate value of 502 times 9"/>
-       </numericalresponse>
-   </p>
-
-   <p>Enter the number of fingers on a human hand:
-       <numericalresponse answer="5">
-           <formulaequationinput label="Enter the number of fingers on a human hand"/>
-       </numericalresponse>
-   </p>
-       <solution>
-           <div class="detailed-solution">
-               <p>Explanation</p>
-               <p>Pi, or the the ratio between a circle's circumference to its diameter, is an irrational number known to extreme precision. It is value is approximately equal to 3.14.</p>
-               <p>Although you can get an exact value by typing 502*9 into a calculator, the result will be close to 500*10, or 5,000. The grader accepts any response within 15% of the true value, 4518, so that you can use any estimation technique that you like.</p>
-               <p>If you look at your hand, you can count that you have five fingers.</p>
-           </div>
-       </solution>
-   </problem>
+        [explanation]
+        The sun is 93,000,000, or 9.3*10^6, miles away from Earth.
+        -100 is the square of 10 times the imaginary number, i.
+        [explanation]
+data: |
+    <problem>
+
+    <p>In a numerical response problem, learners enter numbers or a specific and
```

Comment:
numerical input or numerical response? I think we should be consistent (this occurs in other problem types too)